src/eko/anomalous_dimensions/as4/gNSm.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+"""
+This module contains the anomalous dimension :math:`\\gamma_{ns,-}^{(3)}`
+"""
+import numba as nb
+
+from ...constants import CF
+from ...harmonics.constants import zeta3
+
+
+@nb.njit(cache=True)
+def gamma_ns_nf3(n, sx):
```

Comment:
It is stupid, I know, but since both me @felixhekhorn are now adopting the standard NumPy doc strings layout, can you reformat this one and the others to be compliant with the [Sphinx provided example](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html) please?

In particular, wrt what you usually do:
- the first line, containing a brief description of the function, should be on the same line of the starting `"""`
- no further indentation before variable names in `Parameters` and `Returns` sections (they start on the same column of the header, the description is still indented once wrt the name)
- leave a blank line before the closing `"""`

This way we'll incrementally standardize the format, and reduce the diffs in future.

Reply:
okay fine :)